feat: conversation link button in memory items

krushnarout opened the pull request:
closes #2889

https://github.com/user-attachments/assets/5307ea9a-fbe5-4e4c-9959-61615665c628

beastoin commented:
<img width="1132" height="1015" alt="Screenshot 2025-09-16 at 11 15 37" src="https://github.com/user-attachments/assets/40559369-4e8d-459b-8c51-5e233e1b48d4" />


what is this? review your code before opening the PR, or just mark it as draft if it’s not ready.

@krushnarout

krushnarout commented:
>what is this? review your code before opening the PR, or just mark it as draft if it’s not ready.

@beastoin I mistakenly included changes of a different approach, thank you

beastoin commented:
lgtm @krushnarout

beastoin commented:
reverted, pls resolve https://github.com/BasedHardware/omi/pull/3000

my question, did you see the build error ? 

@krushnarout

krushnarout commented:
>reverted, pls resolve https://github.com/BasedHardware/omi/pull/3000

>my question, did you see the build error ?

No, before every pull request, I use `flutter analyze`. How do I check for build errors ?

@beastoin

krushnarout commented:
before it was working fine

https://github.com/user-attachments/assets/5307ea9a-fbe5-4e4c-9959-61615665c628

after I added the changes from #2988 to the latest `main` branch, I see so many errors, and also it's broken.

https://github.com/user-attachments/assets/085455b4-9f33-49f9-abb8-90d6a3c984ec

fixed in pr #3005

@beastoin
